src/views/layout.njk

```diff
@@ -1,27 +1,13 @@
-{% extends "govuk/template.njk" %}
-
-{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
-{% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
-{% from "govuk/components/button/macro.njk" import govukButton %}
-{% from "govuk/components/date-input/macro.njk" import govukDateInput %}
-{% from "govuk/components/details/macro.njk" import govukDetails %}
-{% from "govuk/components/error-message/macro.njk" import govukErrorMessage %}
-{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
-{% from "govuk/components/footer/macro.njk" import govukFooter %}
-{% from "govuk/components/header/macro.njk" import govukHeader %}
-{% from "govuk/components/input/macro.njk" import govukInput %}
-{% from "govuk/components/panel/macro.njk" import govukPanel %}
+{% extends "govuk/template.html" %}
```

Comment:
It should be `{% extends "govuk/template.njk" %}` here.

Reply:
silly mistake of mine